deps/jerry-v8/src/v8jerry_templates.cpp

```diff
@@ -48,9 +48,9 @@ jerry_value_t JerryV8GetterSetterHandler(
 static void JerryV8GetterSetterHandlerDataFree(void *data_p) {
     JerryV8GetterSetterHandlerData* data = reinterpret_cast<JerryV8GetterSetterHandlerData*>(data_p);
 
-    if (data->is_setter == false && data->external) {
-        jerry_release_value(data->external);
-    }
+    // if (data->is_setter == false && data->external) {
```

Comment:
Hi @zherczeg. I have one question.
When I use v8::External, abort is created because of the line. Can you tell me what the problem is?
I'm not sure I don't need this line.

Reply:
Well, multiple types (Objects, Functions) use this helper, and some of them might need this, others don't. I think you can remove it, and if it leaks we will check it.

Reply:
Thanks for the answer.